op-e2e/README.md

```diff
@@ -0,0 +1,20 @@
+# op-e2e
+
+The end to end tests in this repo depend on genesis state that is
+created with the `bedrock-devnet` package. To create this state,
+run the following commands from the root of the repository:
+
+```bash
+make install-geth
+make devnet-allocs
```

Comment:
Maybe we don't always run `make install-geth`, but we should set the makefile rules up such that `make devnet-allocs` is ran when `devnetL1.json` is newer than anything in `.devnet` & should be ran when running `make test` inside op-e2e. I don't like that we manually have to run several steps to run tests & I think it's likely that we will run into issues where `devnet-allocs` gets out of date.

Reply:
I def see what you mean. I should clarify in the readme that you only need to run `make install-geth` once and then again if we update the version

We need to run `make devnet-allocs` if anything in `contracts-bedrock/{src,deploy-config,scripts}` changes

Reply:
This doesn't work very well with make. 9ed901f58f15e82480c372b2ca8b726de0fefa37 was an attempt at solving this but it wasn't portable to `sh` and making bash the shell broke ci